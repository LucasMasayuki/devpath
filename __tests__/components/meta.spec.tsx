import React from 'react'
import renderer from 'react-test-renderer';
import Meta from '@/components/meta';

describe('Footer', () => {
  it('should render Footer', () => {
    const component = renderer.create(
      <Meta />
    )

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})