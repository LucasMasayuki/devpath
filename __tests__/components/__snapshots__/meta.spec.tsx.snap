// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render Footer 1`] = `null`;
